modules/python/package/cv2/__init__.py

```diff
@@ -179,3 +179,13 @@ def load_first_config(fnames, required=True):
 
 
 bootstrap()
+
+# hotfix for pylint issue (https://github.com/opencv/opencv-python/issues/570)
+from . import getStructuringElement, MORPH_ELLIPSE
+
+# extra imports for a proper autocomplete work in IDE
+from . import _registerMatType
```

Comment:
Users should not use private internal symbols directly.

Reply:
Yes, you're right, it shouldn't be here. I've just faced an error, when `mat_wrapper` module couldn't be imported w/o it. It's been when we have imported `*.so` file as `cv2` module in `opencv-python` package.

Removed it.